live-examples/css-examples/fonts/meta.json

```diff
@@ -34,6 +34,14 @@
             "fileName": "font-weight.html",
             "title": "CSS Demo: font-weight",
             "type": "css"
+        },
+        "font-size": {

```

Comment:
This should be camel-cased - `fontSize` - and appear in alphabetical order in meta.json.